uaaextras/webapp.py

```diff
@@ -506,6 +509,24 @@ def invite():
 
         # if we've reached here we are POST, and they've asked us to invite
 
+        # check our token, and expirary date
+        token = session.get("UAA_TOKEN", None)
+
+        try:
+            decoded_token = g.uaac.decode_access_token(token)
+        except:
+            logging.exception("An invalid access token was decoded")
+            return render_template("error/token_validation.html"), 401
+        
+        user_id = decoded_token["user_id"]
+
+        #check for org_manager role
+        cf_client = CFClient(app.config["CF_API_URL"], app.config["CF_USER"], app.config["CF_PASS"])
+        if not cf_client.is_org_manager(cf_client._get_cf_client(), user_id):
+            logging.info('non-org-manager attempted user invite')
+            flash("You must be an org manager to invite a user.")
```

Comment:
I think we could make this message friendlier, maybe:
```
Only organization managers can invite users. See our documentation here <link>
```

Reply:
Updated